### PR TITLE
minor readme edits

### PR DESCRIPTION
**Pull request by ignasi35**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Note that the source generation for the services/client is all done by the `Akka
 
 ## Running
 
-Run this using [sbt](http://www.scala-sbt.org/). Since we want to run the application using HTTP/2 (which implies HTTPS)
+Since we want to run the application using HTTP/2 (which implies HTTPS)
```
Comment (raboof) [inline README.md]: This sentence looks like it could be simplified further ;). Perhaps mention you do need to have sbt installed?

**Review by raboof — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ignasi35) [reply README.md]: Agreed. How about now?

File: README.md [hunk 2]
```diff
@@ -22,8 +22,7 @@ Note that the source generation for the services/client is all done by the `Akka
 
 ## Running
 
-Run this using [sbt](http://www.scala-sbt.org/). Since we want to run the application using HTTP/2 (which implies HTTPS)
-since gRPC requires the transport to be HTTP/2, we'll use the below helper script:   
+Run this using application requires [sbt](http://www.scala-sbt.org/). gRPC requires the transport to be HTTP/2 so we want to Play to use HTTP/2 (which, in Play, implies HTTPS). We'll use the below helper script:
```
Comment (raboof) [inline README.md]: :+1: - 'Run'->'Running'?

Comment (ignasi35) [reply README.md]: 🤦‍♂️

File: README.md [hunk 3]
```diff
@@ -22,8 +22,7 @@ Note that the source generation for the services/client is all done by the `Akka
 
 ## Running
 
-Run this using [sbt](http://www.scala-sbt.org/). Since we want to run the application using HTTP/2 (which implies HTTPS)
-since gRPC requires the transport to be HTTP/2, we'll use the below helper script:   
+Running this using application requires [sbt](http://www.scala-sbt.org/). gRPC requires the transport to be HTTP/2 so we want to Play to use HTTP/2 (which, in Play, implies HTTPS). We'll use the below helper script:
```
Comment (dwijnand) [inline README.md:25]: More 🤦‍♂️. "running this using application requires"?

Comment (dwijnand) [reply README.md:25]: "we want to Play to use HTTP/2"?

Comment (dwijnand) [reply README.md:25]: https://github.com/playframework/akka-grpc-play-quickstart-scala/pull/9

**Pull request merged**